exercises/pascals-triangle/PascalsTriangle.cs

```diff
@@ -7,4 +7,14 @@ public static IEnumerable<IEnumerable<int>> Calculate(int rows)
     {
         throw new NotImplementedException();
     }
+
+    private static IEnumerable<int> Row(int row)
```

Comment:
Could you remove this function? The stub implementation only needs the public methods to allow it to compile.